### ISSUE TITLE
Expected at least 0 tabs, found 0

### ISSUE THREAD

**Issue by marcinant** [WordPress-Coding-Standards/WordPress-Coding-Standards#357]
I got report with these entries from `phpcs --standard=WordPress :

```
1 | ERROR | [ ] File contains UTF-8 byte order mark, which may corrupt your application
2 | ERROR | [x] Line indented incorrectly; expected at least 0 tabs, found 0
```
I use Ubuntu 14.04 with UTF-8 as default coding system.

1. It should be warning. It's not an error nowadays.

2. No comments: expected 0, found 0

**Comment by JDGrimes** [comment]
> 1. It should be warning. It's not an error nowadays.

A BOM is definitely an error in a code file. If you leave that in you will probably get mysterious "headers already sent" errors, because PHP will output the BOM. 



> 2. No comments: expected 0, found 0

This does look like a bug. Maybe it is related to the previous error? If you remove the BOM, does this error go away? Ultimately though, it isn't our bug, because it is from `Generic.WhiteSpace.ScopeIndent.Incorrect`. You'd need to report it [upstream](https://github.com/squizlabs/PHP_CodeSniffer/issues).

**Comment by marcinant** [comment]
Problem is that you are right. There is no BOM in my file. I cleaned it up with `awk`, `sed` etc and PHPCS is saying it's still there.

**Comment by JDGrimes** [comment]
Oh, that is a problem! The error is coming from `Generic.Files.ByteOrderMark.Found`. Again, this is a sniff included with PHPCS and not one of our custom ones, so you'd need to report the problem to them.
